models/access_test.go

```diff
@@ -47,8 +49,10 @@ func TestHasAccess(t *testing.T) {
 
 	user1 := AssertExistsAndLoadBean(t, &User{ID: 2}).(*User)
 	user2 := AssertExistsAndLoadBean(t, &User{ID: 5}).(*User)
-	repo1 := AssertExistsAndLoadBean(t, &Repository{OwnerID: 2, IsPrivate: false}).(*Repository)
-	repo2 := AssertExistsAndLoadBean(t, &Repository{OwnerID: 3, IsPrivate: true}).(*Repository)
+	// A public repository owned by User 2
+	repo1 := AssertExistsAndLoadBean(t, &Repository{ID: 1}).(*Repository)
```

Comment:
Why did you remove the `IsPrivate: false` part?

Reply:
> [13:16] lunny: JonasFranz because bool has default value false, Don’t know it means false or null
> [13:16] lunny: Sess.Get has the same behavior
> [13:16] JonasFranz: Okay, so we need to check it afterwards?

Since it is not supported to use bools in xorm, I propose to add an check if IsPrivate is set right:

```go
assert.False(t, repo1)
```

Reply:
I changed it such that it really loads the expected repository. The resulting repository is the same, just an assurance.
That's applicable to the other changes in `access_test.go` too.

Reply:
But it would be good to check that the repository is really private/public a comment does not ensure that.

Reply:
I can add it of course, just think that it's nothing that adds value since its predefined by the fixtures anyway.